app/assets/javascripts/avalon.js

```diff
@@ -47,4 +47,6 @@ $(document).ready(function() {
     })
     $('input[readonly], textarea[readonly]').attr("readonly", false);
   }
+
+  $('a').has('img, ul').addClass('block');
```

Comment:
Agreed. This is a fix for browser cross-compatibility - WebKit doesn't expand a link to wrap it's contained content without using `inline-block` or `block`. FF doesn't support `inline-block` the same way as WebKit and using `block` universally on links has poor (visual) consequences.